valhalla/thor/matrix_common.h

```diff
@@ -72,7 +72,9 @@ inline std::string get_date_time(const std::string& origin_dt,
   if (pred_id.Is_Valid()) {
     // get the timezone of the output location
     auto out_nodes = reader.GetDirectedEdgeNodes(pred_id, tile);
-    dest_tz = reader.GetTimezone(out_nodes.first, tile) || reader.GetTimezone(out_nodes.second, tile);
+    dest_tz = reader.GetTimezone(out_nodes.first, tile);
```

Comment:
ah right, this fixes #4360

Reply:
in my defense, python damaged my mind a bit

Reply:
please dont resolve these, its helpful for other reviews :pray: